data/videos.yml

```diff
@@ -3693,3 +3693,30 @@ Seattle Xcoders February 2016:
     title: Continuous Integrations for iOS Apps
     tags: [xcode, build, continuous integration]
     vimeo: 158290425
+NSLondon February 2016:
+  - language: English
+    speakers: [Raimon Lapuente, Gergely Orosz]
+    title: "Sorry Objc: It’s not you, it’s me"
```

Comment:
https://travis-ci.org/chriseidhof/pomotv/builds/122938451#L2552

The build fails because the path generated for this video is `build/events/nslondon/february-2016/gergely-orosz-raimon-lapuente-sorry-objc-it?s-not-you-it?s-me.html`. The curly closing quotation marks, `’`, are being turned into `?`, which not allowed in URL paths.

I don’t know why the name is being processed like this. Compare these two, which have `:` and `’`, and are wrapped in `"`:

    title: "Sorry Objc: It’s not you, it’s me"
    title: "Let’s Play: Refactor the Mega-Controller"

The second is working: it is processed into `build/events/nsspain/2015/andy-matuschak-let’s-play-refactor-the-mega-controller.html`